modules/azure/worker/workers.tf

```diff
@@ -18,7 +18,7 @@ resource "azurerm_storage_account" "tectonic_worker" {
 }
 
 resource "azurerm_storage_container" "tectonic_worker" {
-  name                  = "vhd"
+  name                  = "${var.cluster_name}-worker"
```

Comment:
is it worth to include 'vhd' in the name so its easier to distinguish that this is a disk?